packages/components/src/private-apis.ts

```diff
@@ -28,11 +22,11 @@ import { lock } from './lock-unlock';
 
 export const privateApis = {};
 lock( privateApis, {
-	CompositeV2,
-	CompositeGroupV2,
-	CompositeItemV2,
-	CompositeRowV2,
-	useCompositeStoreV2,
+	CompositeV2: Composite,
+	CompositeGroupV2: Composite.Group,
+	CompositeItemV2: Composite.Item,
+	CompositeRowV2: Composite.Row,
+	useCompositeStoreV2: useCompositeStore,
```

Comment:
Oh, so technically we're not unlocking them yet in this PR? Should we keep a `tags: [ 'status-private' ]` on our story then?

Reply:
Unlocking will happen in a follow-up (as per this [feedback](https://github.com/WordPress/gutenberg/pull/63364#pullrequestreview-2174448617)), but I don't think we should add the private tag because, with this PR, the component is exported from the package and available for consumption without lock/unlock

Reply:
Ah ok, got it 👍